docs/api/js/accounts-l1-l2.md

```diff
@@ -153,7 +153,7 @@ async getBaseCost(params: {
 
 | Name                        | Description                                                                                            |
 | --------------------------- | ------------------------------------------------------------------------------------------------------ |
-| params.l2gasLimit           | The `l2gasLimit` for the call.                                                                          |
+| params.gasLimit             | The `gasLimit` for the call.                                                                          |
```

Comment:
Just a suggestion, but maybe it is worth making it clear when we're talking about L1 and L2 here. I note below that we still say "the gas price of the L1 transaction" and we should make sure we're being as unambiguous as possible here.

Reply:
absolutely agree!
will fix.